backend/djangoindia/api/views/event.py

```diff
@@ -52,3 +52,24 @@ def post(self, request, *args, **kwargs):
             return Response(
                 {"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR
             )
+
+class NewsletterSubscriptionAPIView(generics.GenericAPIView, ListModelMixin, CreateModelMixin, RetrieveModelMixin):

```

Comment:
```suggestion
class NewsletterSubscriptionAPIView(generics.GenericAPIView, CreateModelMixin):

```

Reply:
I got it. I will make modifications asap. Thanks